frontend/components/EmptyState/_styles.scss

```
@@ -0,0 +1,215 @@
+// EmptyState — decorative ghost-table empty state component.
+//
```

Stylelint (17.9.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)